docs/JLBP-14.md

```diff
@@ -1,29 +1,29 @@
 # [JLBP-14] Do not use version ranges
 
-- If the pom.xml of your library specifies a version range instead of a
-  single version for any particular dependency, builds at different points
-  in time can see different versions of that dependency (as your dependency
-  releases new versions), which can break your library unexpectedly.
+Always specify a single version instead of a version range for dependencies.
```

Comment:
I would combine this paragraph with the next one.

Reply:
done